site/docs/numbers/porting/updatePortin.mdx

```diff
@@ -0,0 +1,233 @@
+---
+id: updatePortin
+title: Manage Port-in
+slug: /numbers/guides/porting/updatePortin
+description: How to update your port-in request
+keywords:
+  - bandwidth
+  - numbers
+  - porting
+  - portin
+  - port-in
+  - lnp
+  - port
+  - update
+  - supp
+image: '@site/static/img/bw-icon.svg'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const accountId = "{accountId}";
+export const orderId = "{orderId}";
+
+export const Highlight = ({children, color}) => (
+    <span
+      style={{
+        backgroundColor: color,
+        borderRadius: '2px',
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+In this guide we will show you how to manage your Port in order. We assume that you have [created Port in order](/docs/numbers/guides/porting/updatePortin).
```

Comment:
Change "manage" to "update" 
Change "Port in" to "port-in"

Reply:
I am wandering which form is correct, "Port in" for headers and "port-in" for occurrences in text?

Reply:
In headers, keep "Port-in" (with the dash) and "port-in" in other places in text.